### PR TITLE
Make \pdfpcnote command overlay-aware

### LINKED ISSUE #9: Make `\pdfpcnote` overlay-specification aware
Currently, the provided `\pdfpcnote` (and, when `overridenote` is set, `\note`) commands are not compatible with Beamer's overlay specifications. Beamer documentation explicitly recommends using overlays with their built-in `\note` command, so similar support in the pdfpc package would be greatly appreciated.

Additionally, implementing this support should be extremely straightforward — probably requiring just one extra `\only` in the relevant command definitions. I've experimented *very slightly* with such redefinitions myself, and given the apparent ease of implementation, would be willing to PR this feature if desired.

(Section 9.2 of the Beamer manual should be relevant, if additional detail is needed.)

### PR DESCRIPTION
**Pull request by LRitzdorf**
Adds support for Beamer overlay specifications to the provided `\pdfpcnote` command.

Resolves #9

### CODE REVIEW

**Comment by LRitzdorf** [comment]
Should be ready to merge, pending developer approval! I've tested my changes locally, and they do appear to work as intended.

**Comment by AndreasBilke** [comment]
Apparently I am not a LaTeX macro expert so I can't verify your solution. But: Could you also extend the pdfpc package documentation about this new feature?

**Comment by LRitzdorf** [comment]
Sure! I'll take care of that shortly.

In the meantime, if you do want to test this, you should be able to simply place the new `.sty` file at a LaTeX project root and compile.

**Review by fnevgeny — APPROVED** [review]

**Pull request merged**
